translate.yml

```diff
@@ -0,0 +1,2 @@
+greeting:
```

Comment:
Delete this file. We don't need it.

At a later stage, we'll add examples but not now.